app/controllers/users_controller.rb

```diff
@@ -1,11 +1,11 @@
 class UsersController < ApplicationController
-  def index; end
-
-  def show
-    @id = params[:id]
+  def index
+    @users = User.all
   end
 
-  def posts
-    @id = params[:user_id]
+  def show
+    user_id = params[:id].to_i
+    @user = User.find(user_id)
+    @posts = @user.recent_posts
```

Comment:
- [ ] When opening the user#show page, I get an error related to recent_posts. I checked and you have a `latest_posts` method in the user mode but no `recent_posts`. So kindly fix it 
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/37901469/230077220-769ea335-43b2-4651-910b-3ded116fee88.png">

**You can do it**